Sources/Fullscreen/Explore/ExploreView.swift

```diff
@@ -29,6 +29,27 @@ public protocol ExploreViewDataSource: AnyObject {
     )
 }
 
+public protocol ExploreViewRecommendationsDatasource: AnyObject {
+    func exploreViewRecommendations(_ exploreView: ExploreView, cellClassesIn collectionView: UICollectionView) -> [UICollectionViewCell.Type]
+    func exploreViewRecommendations(_ exploreView: ExploreView, collectionView: UICollectionView, cellForItemAt indexPath: IndexPath) -> UICollectionViewCell
+}
```

Comment:
Do we need these methods? I see `StandardAdRecommendationCell`, and other cells, are still registered when creating the `collectionView`.

Reply:
We need them to register AfshBannerCollectionViewCell and AppNexusNativeCollectionViewCell that exist in the finn app, I guess I can remove the registration of the StandardAdRecommendationCell when I create the collection view for consistency, so all recommendation cells are registered in setup?